cmd/config/fs_config.go

```diff
@@ -98,6 +98,26 @@ func (c FSConfig) AliasEnvironment(url, alias, caCert string) (Config, error) {
 	return config, nil
 }
 
+func (c FSConfig) DeleteAlias(alias string) (Config, error) {
```

Comment:
being picky... can we rename this to UnaliasEnvironment to be consistent with AliasEnvironment.

Reply:
Sure.  I only left this one because from the perspective of the config we are actually _deleting_ the alias.